pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
+line-length=79
```

Comment:
Black defaults to 88 characters per line. I've changed it back to 79, which is the PEP8 recommended value. As explained above, Black might still go over the maximum length in some cases.